com.arc.embeddedcdt/src/com/arc/embeddedcdt/LaunchConfigurationConstants.java

```diff
@@ -58,6 +59,11 @@
     String ATTR_FTDI_DEVICE = LAUNCH_ID + ".ftdi_device"; //$NON-NLS-1$
     String ATTR_FTDI_CORE = LAUNCH_ID + ".ftdi_core"; //$NON-NLS-1$
 
+    static final int UNREAL_DEBUGGER_PLUGIN_VERSION_NUMBER = -1;
```

Comment:
Could be made shorted, no need to repeat DEBUGGER, since you are in debugger plugin anyway. Something like `UNREAL_FILE_FORMAT_NUMBER`.